pkg/controller/storagecluster/reconcile.go

```diff
@@ -1190,6 +1190,11 @@ func newCleanupJob(sc *ocsv1.StorageCluster) *batchv1.Job {
 		"app": "ceph-toolbox-job-${FAILED_OSD_ID}",
 	}
 
+	const purgeOSDScript = `set -x
```

Comment:
You're missing the line from the [script](https://gist.github.com/travisn/782cf2ef9f4956d7799d658fca12942c) that sets `$osd_status`.

Reply:
yes